Rocket.Chat.ShareExtension/Resources/de.lproj/Localizable.strings

```diff
@@ -0,0 +1,32 @@
+/* 
+  Localizable.strings
+  Rocket.Chat
+
+  Created by Matheus Cardoso on 2/28/18.
+  Copyright © 2018 Rocket.Chat. All rights reserved.
+*/
+
+"global.ok" = "Ok"; //TODO
+
+"alert.no_servers.title" = "No servers"; //TODO
+"alert.no_servers.message" = "You need to join at least one Rocket.Chat server in order to share something."; //TODO
+
+"servers.title" = "Servers"; //TODO
+
+"rooms.title" = "Select Rooms"; //TODO
+"rooms.section.server" = "Select server"; //TODO
+"rooms.section.favorites" = "Favorites"; //TODO
+"rooms.section.channels" = "Channels"; //TODO
+"rooms.section.groups" = "Private Groups"; //TODO
+"rooms.section.directMessages" = "Direct Messages"; //TODO
+
+"compose.title" = "Share"; //TODO
+"compose.send" = "Send"; //TODO
+"compose.destination.to" = "To: "; //TODO
+"compose.file.name.placeholder" = "File name"; //TODO
+"compose.file.description.placeholder" = "File description"; //TODO
+
+"report.success.title" = "Success"; //TODO
+"report.success.message" = "Shared your content successfully."; //TODO
+"report.error.title" = "Error"; //TODO
+"report.error.message" = "Something wrong happened while sending your content."; //TODO
```

Comment:
Any chance we get a help from our German speakers friends? (cc @soundstorm, @artrmz) :-)

Reply:
@rafaelks Since it's @soundstorm's native language I'll leave it to him and helped with polish translation instead  ;)

Reply:
Thanks @artrmz!!!

Reply:
/* 
  Localizable.strings
  Rocket.Chat

  Created by Matheus Cardoso on 2/28/18.
  Copyright © 2018 Rocket.Chat. All rights reserved.
*/

"global.ok" = "Ok";

"alert.no_servers.title" = "Keine Server";
"alert.no_servers.message" = "Es wird mindestens ein eingerichteter Rocket.Chat Server benötigt, um etwas zu teilen.";

"servers.title" = "Server";

"rooms.title" = "Raum auswählen";
"rooms.section.server" = "Server auswählen";
"rooms.section.favorites" = "Favoriten";
"rooms.section.channels" = "Kanäle";
"rooms.section.groups" = "Private Gruppen";
"rooms.section.directMessages" = "Direktnachrichten";

"compose.title" = "Teilen";
"compose.send" = "Senden";
"compose.destination.to" = "An: ";
"compose.file.name.placeholder" = "Dateiname";
"compose.file.description.placeholder" = "Beschreibung";

"report.success.title" = "Erledigt";
"report.success.message" = "Die Inhalt wurde erfolgreich versandt.";
"report.error.title" = "Fehler";
"report.error.message" = "Etwas ist schiefgelaufen beim Teilen.";

Reply:
@cardoso :)